datalake/src/main/resources/application.yml

```diff
@@ -33,8 +35,14 @@ datalake:
   structuredevent:
     rest:
       enabled: false
+  cloudbreak:
+    url: http://localhost:9091
```

Comment:
please add pathContext as well like in cloudbreak or environment services

Reply:
not needed in sdx, it will use default value:  https://github.com/hortonworks/cloudbreak/blob/826f6466d4d11b2881a16094af138cdba900298b/datalake/src/main/java/com/sequenceiq/datalake/configuration/ServiceEndpointConfig.java#L42

Reply:
it is a little bit confusing. I think it is more readable if we handle in one place

Reply:
this is the way how we doing it across services